src/Chainweb/Miner/Config.hs

```diff
@@ -68,20 +73,34 @@ newtype MinerCount = MinerCount { _minerCount :: Natural }
 -- -------------------------------------------------------------------------- --
 -- Mining Config
 
-validateMinerConfig :: ConfigValidation MiningConfig []
-validateMinerConfig c = do
-    when (_nodeMiningEnabled nmc) $ tell
-        [ "In-node mining is enabled. This should only be used for testing"
-        , "In order to use in-node mining, mining-coordination must be enabled, too"
-        ]
-    when (_nodeMiningEnabled nmc && not (_coordinationEnabled cc))
-        $ throwError "In-node mining is enabled but mining coordination is disabled"
-    when (_nodeMiningEnabled nmc && view minerId (_nodeMiner nmc) == "")
```

Comment:
Is this check also relevant when just mining coordination is enabled (but no node-mining)?